src/main/java/io/hops/metadata/ndb/dalimpl/hdfs/QuotaUpdateClusterj.java

```diff
@@ -138,6 +140,7 @@ public void prepare(Collection<QuotaUpdate> added,
       }
       if (added != null) {
         for (QuotaUpdate update : added) {
+          LOG.error("Added quota update "+update);
```

Comment:
Should this be here?

Reply:
no, it should not be there. I have removed it. Thanks.